index.md

```diff
@@ -13,6 +13,17 @@ A tool that accelerates the process of re-platforming to Kubernetes by analyzing
 
 move2kube is a command line tool that uses source artifacts such as docker compose files or cloud foundry manifest files, and even source code to generate kubernetes deployment artifacts including object yaml, helm charts, and operators.
 
+## Usage
+
+Running Move2Kube is a simple single-step process. Move2Kube takes as input the source artifacts and outputs the target deployment artifacts.
```

Comment:
its not always a single step process. It can be used as a multi-stage process too. Lets think through the right way to bring this out.

Reply:
Yes sure.